docs_src/general/ServicePorts.md

```diff
@@ -4,16 +4,19 @@ The following tables (organized by type of service) capture the default service
 === "Core"
     |Services Name|	Port Definition|
     |---|---|
-    |core-data|	59880|
-    |ZMQ - to be deprecated in a future release|5563|
+    |core-data|	59880, 5563 (ZMQ; deprecated)|
```

Comment:
This is confusing for beginning users. They might think the core data is deprecated, or ZMQ uses both ports.
How about:
```
    |core-data                                      |59880|
    |core-data (for deprecated ZMQ)    |5563|
```
or
```
|core-data                 | 59880 (general), 5563 (ZMQ; deprecated)|
```

Reply:
Agreed. Added a line break.

I'm actually not sure why we need to specify here that it is deprecated. I'm sure there is a page somewhere documenting what is being deprecated.